docs/references/backend/organization/get-organization-list.mdx

```diff
@@ -20,9 +20,9 @@ function getOrganizationList: (params: GetOrganizationListParams) => Promise<Pag
 | `query?` | `string` | A search query to filter organizations by. |
 | `orderBy?` | `name \| updated_at \| members_count` | The field to order by. Prefix with a `-` to reverse the order. Prefix with a `+` to list in ascending order. |
 
-## `getOrganizationList()` examples
+## Examples
 
-### `getOrganizationList()` with no parameters
+### Example with no parameters
```

Comment:
I'm realizing pages like this will clutter the search results as well, because there are multiple headings with `getOrganizationList` in them. I don't think we should address that in this PR, because it would require the more in depth work of changing each example's heading to describe what the example does rather than show which param is being passed.

I added an issue for this to triage: https://linear.app/clerk/issue/DOCS-6264/update-example-headings-to-describe-what-the-examples-are

Reply:
No, we'd simplify the sentence we have under each of the headings. So for [your example](https://clerk.com/docs/references/backend/organization/get-organization-invitation-list#get-organization-invitation-list-organization-id-status), it says:

> Retrieves organization invitation list that is filtered by the status of the invitation.

We could rename the heading to something as simple as "Filter by invitation status". Just something that puts what the example does in simple terms. I think users would find that a more useful thing to see in the ToC as well